### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by talexandru1987**
Completed application

### CODE REVIEW

File: assets/js/index.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+const handleClickEvent = (event) => {
+  //get the event target
+  const target = event.target;
+  // get data attribute value
+  const textAreaID = target.dataset.button;
+
+  //get the index in the object
+  const objectIndex = target.dataset.index;
+
+  console.log(objectIndex);
```
Comment (LeeFarnell) [inline assets/js/index.js]: Please remember to remove your console logs.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    </header>
+    <div class="container">
+      <!-- Timeblocks go here -->
+      <!-- <div id="container-flex-div">
+        <div id="span-div">
+          <span id="time-span" class="hour-nine">9AM</span>
+        </div>
+        <textarea name="" id="event-text" cols="30" rows="10"></textarea>
+        <button id="container-button" class="fa fa-lock"></button>
+      </div> -->
```
Comment (LeeFarnell) [inline index.html]: Please remove all unused code as this makes code look messy.

File: assets/js/index.js (hunk 1 above)
Comment (talexandru1987) [reply assets/js/index.js]: Done

File: index.html (hunk 2 above)
Comment (talexandru1987) [reply index.html]: Done

**Pull request merged**